src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -48,10 +48,12 @@ const style = {
 };
 
 export const NetworkStatusBar = () => {
-  const getStatusText = (networkSettings) => `${customNetworkText}: ${networkSettings.network}`;
+  const getStatusText = (networkName) => `${customNetworkText}: ${networkName}`;
   const networkSettings = useSelector((state) => state.networkSettings);
+  const networkName = useSelector((state) => state.networkName);
```

Comment:
suggestion(non-blocking): Create the function only after declaring its dependency variables. Also, fetch all `store` data in a single selector fetch.

```js
const { networkSettings, networkName } = useSelector((state) => ({
  networkSettings: state.networkSettings,
  networkName: state.networkName,
});
const getStatusText = (networkName) => `${customNetworkText}: ${networkName}`;
```